### DISCUSSION TITLE
New user - download csv?

### DISCUSSION THREAD

**Discussion by QuinolKO** [community/community#23348] [New to GitHub]
<p>hi -</p>
<p>im trying to download this file as .csv. can anyone give some tips to save a csv to my desktop?</p><aside class="onebox githubblob" data-onebox-src="https://github.com/nytimes/covid-19-data/blob/fc30edc42780f56fda80cde595567716526c991e/us-states.csv">
  <header class="source">

      <a href="https://github.com/nytimes/covid-19-data/blob/fc30edc42780f56fda80cde595567716526c991e/us-states.csv" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/nytimes/covid-19-data/blob/fc30edc42780f56fda80cde595567716526c991e/us-states.csv" target="_blank" rel="noopener nofollow ugc">nytimes/covid-19-data/blob/fc30edc42780f56fda80cde595567716526c991e/us-states.csv</a></h4>

    <pre><code class="lang-csv">date,state,fips,cases,deaths
2020-01-21,Washington,53,1,0
2020-01-22,Washington,53,1,0
2020-01-23,Washington,53,1,0
2020-01-24,Illinois,17,1,0
2020-01-24,Washington,53,1,0
2020-01-25,California,06,1,0
2020-01-25,Illinois,17,1,0
2020-01-25,Washington,53,1,0
2020-01-26,Arizona,04,1,0
2020-01-26,California,06,2,0
2020-01-26,Illinois,17,1,0
2020-01-26,Washington,53,1,0
2020-01-27,Arizona,04,1,0
2020-01-27,California,06,2,0
2020-01-27,Illinois,17,1,0
2020-01-27,Washington,53,1,0
2020-01-28,Arizona,04,1,0
2020-01-28,California,06,2,0
2020-01-28,Illinois,17,1,0
</code></pre>

  This file has been truncated. <a href="https://github.com/nytimes/covid-19-data/blob/fc30edc42780f56fda80cde595567716526c991e/us-states.csv" target="_blank" rel="noopener nofollow ugc">show original</a>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>thanks in advance for your help!!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The “Raw” button on the right should help you there:<br>
<img src="https://user-images.githubusercontent.com/5814919/181076453-94ec34da-a119-4899-a048-373cd5e816b6.png" alt="image" data-base62-sha1="wYrD6VeLYvmNhSqM0nK23cqd0xk" width="398" height="86"></p>
<p>If you click it you’ll get just the file, without any surrounding UI. Or you could right-click the button and choose the “save as…” option. I hope that helps! :slight_smile:</p>

**Comment by usuargithub** [comment]
<p>Hello,<br>
I think, that there are much more interesting thinks that the SARS-COV-2</p>
<p>Regards</p>
